test/declarative_config/configurations/invalid-ssl-factory.json

```diff
@@ -3,7 +3,7 @@
         "ssl": {
             "enabled": true,
             "factory": {
-                "exportedName": "BsicSSLOponsFary",
+                "exportedName": "MySSLOptionsFactory",
```

Comment:
as a minor improvement, we could name this as something like `NonExistingOptionsFactory` to communicate that this factory name is intended to be wrong.